### DISCUSSION TITLE
font not working

### DISCUSSION THREAD

**Discussion by M4RCOE** [dompdf/dompdf#3272] [Q&A] [answered]
**In dompdf v2.0.0, the font-family change works, but in version v2.0.3, it doesn't.**
```
<style>

   @font-face {
        font-family: "Montserrat";
        font-weight: normal;
        font-style: normal;
        src: url('<?=baseUrl('/assets/fonts/Montserrat-Regular.ttf')?>') format('truetype');
    }
    @font-face {
        font-family: "Montserrat";
        font-weight: bold;
        font-style: normal;
        src: url('<?=baseUrl('/assets/fonts/Montserrat-Bold.ttf')?>') format('truetype');
    }
    :root, html {
        font-family: Montserrat, Arial, Helvetica, sans-serif;
        line-height: 1em;
        font-size: 10pt;
        text-align: justify;
        margin: 0px;
        padding: 0px;
    }

</style>
<?php
   $dompdf = new Dompdf();
   $dompdf->setOptions(new Options(['dpi' => '300']));
   $dompdf->setPaper('letter', 'portrait');
   $dompdf->addInfo('Title', $pdfTitle);
   $dompdf->loadHtml(ob_get_clean());
   $dompdf->render();
   $canvas = $dompdf->getCanvas();
   $pageWidth = $canvas->get_width() / 2;
   $pageHeight = $canvas->get_height() - 30;
   $canvas->page_text($pageWidth, ($pageHeight), "{PAGE_NUM} de {PAGE_COUNT}", null, 8, array(0,0,0));
   $dompdf->stream($pdfTitle, ['Attachment' => false]);
```

**Comment by bsweeney** [comment]
This may be due to the following change in 2.0.1:

> Improved font-face declaration parsing and handling. External fonts are now restricted by resource access constraints.

If you modify your options to the following does the font load?

```
$dompdf->setOptions(new Options(['dpi' => '300', 'isRemoteEnabled' => true]));
```

(edited to specify the correct option name)

**Reply by M4RCOE** [reply]
That configuration doesn't work, I've already tried it.

**Reply by bsweeney** [reply]
Is baseUrl a URL, e.g. http://example.com? If so, what URL does that produce? If the URL starts with HTTP and your site redirects from HTTP to HTTPS that could also be the source of the problem.

FYI, you can try enabling additional debug messaging to try and get to the source of the issue:
https://github.com/dompdf/dompdf/wiki/Debugging-Dompdf

**Reply by M4RCOE** [reply]
render in HTML in browser show the font but the render in PDF font not working
My URL is: http://localhost:30005/assets/fonts/Montserrat-Regular.ttf
this configuration https://github.com/dompdf/dompdf/wiki/Debugging-Dompdf doesn't show anything

**Reply by bsweeney** [reply]
Do you have a sample of the output?

**Reply by M4RCOE** [reply]
![EXAMPLE](https://github.com/dompdf/dompdf/assets/58351754/08385e19-cde3-4449-8634-d5497202a52e)

**Reply by bsweeney** [reply, marked as answer]
Looks like I provided the wrong option name in my earlier suggestion. I specified `isRemoteAllowed` but the option is named `isRemoteEnabled`. Try the following:

```
$dompdf->setOptions(new Options(['dpi' => '300', 'isRemoteEnabled' => true]));
```

(I edited my previous comment to prevent confusion for others)

**Reply by kirill009** [reply]
Hi, I have the same problem and 'isRemoteEnabled' didn't help.

**Reply by bsweeney** [reply]
@kirill009 would need to see more details about your specific document to be able to provide any guidance.
